### ISSUE TITLE
Fission Products 

### ISSUE THREAD

**Issue by Catwar234** [Nomi-CEu/Nomi-CEu#1088]
### Nomi CEu Version

1.7.3

### Launcher

Curse forge 

### Environment

Singleplayer 

### Mode

Normal Mode

### Related Problem

The nuclear fuel chain does not feel "greggy" enough. It lacks the depth and complexity of other lines and chains around the same point in the pack. Things like PBI, PBS, etc. Put in one fuel and get out products for other fuels in its own closed loop. All you really need is the reactor and a centrifuge. Also an influx of a thorium or uranium.

### Your Solution

Two suggestions. One, add something called "Fission Products" this comes in small amounts from depleted fuel (none of the depleted fuels give out more than four things rn so there is space) This could work like rare metal Earth and metal mixture dust, a small chance for some rarer materials back (such as some noble gasses, cesium etc). It could also be a source of other materials that would up the devs. This could add a optional thing for players to do with their fuel line. Second thing, is making the recycling more complex. Instead of just throwing it in a centrifuge, it comes out hot and you have to cool it, then breaking it apart with acids. 

### How Will Your Solution Work

The "Fission Products" was based upon real life Fuel Rod recycling, in real life it contains stuff xenon, krypton, iodine, cesium, and strontium. This adds realism and complexity to a system that is more or less two block solution at the moment. 

### Additional Information

I was super excited to start a new complex chain of processing, but realized it could all be done with a centrifuge and the reactor itself, so I felt a little disappointed (the rest of the pack has been awesome btw it has become my favorite pack of all time)

**Comment by Folanlron** [comment]
because its not the greg fission system..

it is NuclearCrafts fission system.

CEu is still working on it's fission machines and elements. It is coming but even if it was released next week they are still a long way from gutting NC and replacing everything with Gregtechs version.

**Comment by Catwar234** [comment]
Yes, the items are from nuclearcraft but the recipes are specific to this modpack. My point was it lacked the essence of Gregtech, that is why I stopped playing other packs (one block solutions). I did not know however that CEu was making their own version, so that makes me glad thank you for the big of info.

**Comment by IntegerLimit** [comment]
This could be an interesting idea, perhaps some elements can be incorporated into CEu's nuclear if we wanted to make some parts more complex (HM).
